tests/workflows/unit_tests/core_steps/classical_cv/test_distance_measurement.py

```diff
@@ -0,0 +1,455 @@
+import numpy as np
```

Comment:
general comment - assertions on floats are to be made up to specific tolerance, not to the direct values so
```python
assert abs(value - reference) < 1e-5
```
instead of:

```python
assert value == 0.00227276553
```

Reply:
Done